2-alcance/README.md

```diff
@@ -1,3 +1,187 @@
-# Plano de alcance
+# PLANO DE ALCANCE
 
-## Referencias
\ No newline at end of file
+## INTRODUCCIÓN
+
+En el siguiente apartado, se va a especificar la estructura del alcance que tendrá el sitio web. El contenido se dividirá en dos fragmentos principales, como son las especificaciones funcionales y los requerimientos de contenido.
+
+## ESPECIFICACIONES FUNCIONALES
+Las especificaciones funcionales hacen referencia al comportamiento de, en este caso, una página web relacionada con la igualdad de género. La página web se llama Hoyportimañanapormi.com y en ella se encuentra contenido de todo tipo: noticias, reportajes, entrevistas, artículos, historias, consejos, etc.
+
+## HOYPORTIMAÑANAPORMI.COM
+> **Especificación Funcional**
+
+> Petunia Flores
+
+> *Última actualización: 4 de noviembre, 2018*
+                  
+> **- CONFIDENCIAL -**
+
+### 1. GENERALIDADES
+*Hoyportimañanapormi.com* es un servicio, vía web, que ofrece contenido informativo y educativo sobre la igualdad de género. En el portal web se encontrarán noticias, reportajes, entrevistas, etc., y todas ellas relacionadas con la igualdad de género. 
+
+El sitio web necesitará un usuario gestor de contenidos, es decir un usuario administrador que se encargará de actualizar todo el contenido de la página. Por otro lado, existirá un usuario voluntario que trata de aquellos usuarios que quieran formar parte de este proyecto y quieran proporcionar información interesante relacionada con la igualdad de género. Por supuesto esa información tendrá que pasar por la supervisión del usuario administrador; el cual decidirá añadir o no el contenido. Y finalmente, se encuentra el usuario no registrado que será aquel que entre al sitio web y que consuma el contenido sin tener que registrarse. 
+
+La pagina web estará divida en dos partes: la información estática (contacto, quiénes somos, log-in)  y la información dinámica (noticias, articulos, reportajes, entrevistas, historias reales, consejos).
+
+### 2. ESCENARIOS
+En este apartado se han creado tres escenarios supuestos donde se imagina el uso que le pueden dar los usuarios del sitio web. A continuación se encuentran los siguientes escenarios:
+
+#### 2.1. Escenario 1: Paula y su desconocimiento
+Paula tiene 43 años, es directora informática de BOSCH Bilbao pero no todo es perfecto. Se encuentra dentro de una relación sentimental un tanto tóxica o eso empieza a sentir ella. No se atreve a hablar con su entorno sobre las situaciones que está viviendo con su actual pareja. Y como no tiene donde refugiarse decide sumergirse en Internet. En una de sus redes sociales (Facebook), allí encuentra una publicación que ha compartido una amiga suya y que trata sobre un sitio web donde tratan contenido relacionado con la igualdad de género; pero sobre todo es un lugar de refugio, ayuda y consejo. En consecuencia, Paula decide hacer clic en el enlace y entrar en hoyportimañanapormi.com. 
```

Comment:
Ya habéis descrito algunos rasgos de Laura en el apartado anterior; en lugar de repetirlos, podéis hacer referencia a ellos aquí.